internal/engine/fake_notify_test.go

```diff
@@ -0,0 +1,103 @@
+package engine
+
+import (
+	"sync"
+
+	"github.com/windmilleng/tilt/internal/watch"
+)
+
+type fakeMetaWatcher struct {
```

Comment:
Prior to using this meta watcher we were only sending events to one watch, not all the watches that the watch manager had set up.

Reply:
would it be better to break this change out into a separate pr?

Reply:
Yeah that's a good idea.